python/cudf/cudf/utils/nvtx_annotation.py

```diff
@@ -17,12 +19,13 @@ def _get_color_for_nvtx(name):
 
 
 def _cudf_nvtx_annotate(func, domain="cudf_python"):
```

Comment:
Should we rename `_cudf_nvtx_annotate` to something like `_performance_tracking` ?

Reply:
I think that would be a more generically clear name. +1